source/includes/connection-snippets/x509-snippets/python-connection.py

```diff
@@ -0,0 +1,13 @@
+# begin x509 connection
+from pymongo import MongoClient
+
+uri = "mongodb+srv://<cluster-url>/test?authSource=$external&retryWrites=true&w=majority&authMechanism=MONGODB-X509"
+client = MongoClient(uri,
+                     tls=True,
+                     tlsCertificateKeyFile='/etc/certs/mongodb/client.pem')
+
+db = client['testDB']
+collection = db['testCol']
+docCount = collection.count_documents({})
```

Comment:
`doc_count` would be more idiomatic in Python.

Reply:
Fixed